### ISSUE TITLE
Can't build 1.14.0 on Nvidia Xavier AGX

### ISSUE THREAD

**Issue by andreyhristov** [tensorflow/tensorflow#31982]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04, 
- Device : Nvidia Xavier AGX
- TensorFlow installed from (source or binary): source
- TensorFlow version: 1.14.0
- Python version: 3.6
- Installed using virtualenv? pip? conda?: n/a
- Bazel version (if compiling from source): 0.24.1
- GCC/Compiler version (if compiling from source): 7.4.0
- CUDA/cuDNN version: 10.0.326-1
- GPU model and memory: GV10B, 16GB shared

The build process can't start out of the box. Problem is missing instructions for the toolchain.
The following two patches are needed to be able to build:
```
--- third_party/toolchains/cpus/arm/BUILD~      2019-06-19 00:48:23.000000000 +0200
+++ third_party/toolchains/cpus/arm/BUILD       2019-08-26 16:49:33.597480129 +0200
@@ -13,6 +13,7 @@
         "k8": ":cc-compiler-local",
         "piii": ":cc-compiler-local",
         "arm": ":cc-compiler-local",
+        "aarch64": ":cc-compiler-local",
         "s390x": ":cc-compiler-local",
     },
 )
```
and
```
--- third_party/gpus/crosstool/BUILD.tpl~       2019-06-19 00:48:23.000000000 +0200
+++ third_party/gpus/crosstool/BUILD.tpl        2019-08-26 16:50:25.987292816 +0200
@@ -29,6 +29,7 @@
         "x64_windows|msvc-cl": ":cc-compiler-windows",
         "x64_windows": ":cc-compiler-windows",
         "arm": ":cc-compiler-local",
+        "aarch64": ":cc-compiler-local",
         "k8": ":cc-compiler-local",
         "piii": ":cc-compiler-local",
         "ppc": ":cc-compiler-local",
```
It would be nice if these get incorporated into the mainline. Probably related to #22629

**Comment by ravikyram** [comment]
@andreyhristov 

Please,have a look on @mihaimaruseac's suggestion .Thanks!

**Comment by andreyhristov** [comment]
Hi,
I am currently on vacation and don't have access to the hardware

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by andreyhristov** [comment]
Seems has already been fixed in 1.15-rc2
